/**/filter:
  ros__parameters:
    signal_topic: sample_wave
    config_path: ./src/ros2_dsp_filters/filter_signal/filter_config
    config_file_name: float32